plugins/pencilblue/templates/angular/admin/content/media/manage_media.html

```diff
@@ -33,8 +33,25 @@
 			sortDesc: false
 		}];
 
+		$scope.setItem = function(item) {
+				$scope.topic = item;
+				$scope.search();
+				$scope.$apply();
```

Comment:
Shouldn't need a $scope apply for this to work. Two way data binding should cover it.